### PR TITLE
Added a reference to the AUR package built for Arch Linux

### PR DESCRIPTION
**Pull request by frol**
I created an AUR package for cargo-workspaces: https://aur.archlinux.org/packages/cargo-workspaces/

### CODE REVIEW

**Comment by pksunkara** [comment]
How is it built?

**Comment by frol** [comment]
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=cargo-workspaces

```sh
build() {
  cd "${pkgname}-${pkgver}/${pkgname}"
  cargo build --release
}

check() {
  cd "${pkgname}-${pkgver}/${pkgname}"
  cargo test --release
}


package() {
  cd "${pkgname}-${pkgver}/${pkgname}"
  install -Dm755 \
    "target/release/${pkgname}" \
    -t "${pkgdir}/usr/bin"
  install -Dm644 README.md -t "${pkgdir}/usr/share/doc/${pkgname}"
  install -Dm644 ../LICENSE -t "${pkgdir}/usr/share/licenses/${pkgname}"
}
```

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ But this will also work on single crates because by default every individual cra
 
 ```
 cargo install cargo-workspaces
+
+# Or, if you're an Arch user
+yay -S cargo-deny
```
Comment (pksunkara) [inline README.md:30]:
```suggestion
yay -S cargo-workspaces
```

**Comment by pksunkara** [comment]
* Is this part of official aur packages?
* Is this automatically updated?

**Comment by frol** [comment]
It is published to AUR (Arch User Repository). I submit the script and going to maintain it (I watch this repo releases)

**Comment by pksunkara** [comment]
I am afraid that I can't merge this PR unless there is an automated way setup to publish the releases to AUR.

**Comment by pksunkara** [comment]
You can see in this tool, I have an automated way to push releases to homebrew. https://github.com/termapps/enquirer/blob/master/.github/workflows/release.yml

I would like to setup something similar for AUR too.

**Comment by frol** [comment]
I see. I don't have the bandwidth to work on automation, so let me know once there is the release process ready. I am fine with transferring the AUR package to you. Meanwhile, since I use cargo-workspaces myself, I am going to maintain the AUR package.

The maintenance is essentially just:

1. bump the version in the PKGBUILD file
2. update the sha512 checksum: `sed -i "s/sha512sums=.*/$(makepkg -i -g)/" PKGBUILD`
3. build and install locally: `makepkg -i`
4. update src info: `makepkg --printsrcinfo > .SRCINFO`
5. `git add PKGBUILD .SRCINFO && git commit -m "release X" && git push`

**Comment by pksunkara** [comment]
@frol I would love to setup the process. Can you tell me if anyone can submit an AUR package? And can I make a package which distributes the bin instead?

**Comment by frol** [comment]
Yes, anyone can submit a package to AUR. If you want to distribute a binary through the package, name it with -bin suffix convention (https://unix.stackexchange.com/questions/543013/whats-the-difference-between-pkg-pkg-bin-and-pkg-git-on-the-aur). More details about AUR: https://wiki.archlinux.org/index.php/Arch_User_Repository

**Pull request closed without merging**